sysutils/restic/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/restic/restic 0.9.4 v
+categories          sysutils
+
+license             BSD
+maintainers         nomaintainer
+
+description         Fast, secure, efficient backup program.
+
+long_description    Restic is a program that does backups right. Its design goals are: Easy, Fast, Verifiable, Secure, Efficient and Free.
+
+distname            restic-${version}
+homepage            https://restic.net/
+
+checksums           rmd160  05dde2739f3c2ed9109f1ece2e96008cec36ac2a \
+                    sha256  9198c43abd08b3a09ea59226282447316e13da579713dda2d81a28c37902d2c8 \
+                    size    26211387
+
+build.cmd           go run build.go
+build.env-append    GOFLAGS=-mod=vendor
```

Comment:
`-mod=vendor` is already added by build.go, is this GOFLAGS necessary?

https://github.com/restic/restic/blob/v0.9.4/build.go#L572-L577